src/altinn-app-frontend/src/shared/resources/appLanguage/fetch/fetchAppLanguagesSagas.ts

```diff
@@ -0,0 +1,22 @@
+import { SagaIterator } from 'redux-saga';
+import { call, put, take } from 'redux-saga/effects';
+import { get } from 'src/utils/networking';
+import { appLanguagesUrl } from 'src/utils/appUrlHelper';
+import AppLanguagesActions from '../appLanguagesActions';
+import { appTaskQueueError } from '../../queue/queueSlice';
+import { FETCH_APP_LANGUAGES } from './fetchAppLanguagesActionTypes';
+
+function* fetchAppLanguages(): SagaIterator {
+  try {
+    const resource = yield call(get, appLanguagesUrl());
+    yield call(AppLanguagesActions.fetchAppLanguagesFulfilled, resource);
+  } catch (error) {
+    yield call(AppLanguagesActions.fetchAppLanguagesRejected, error);
+    yield put(appTaskQueueError({ error }));
```

Comment:
Maybe instead of crashing the site (as I expect happens by putting this error) you can stop the combobox from being instantiated?
That way language support can be toggled based on actual availability.